internal/cmd/config/set/set.go

```diff
@@ -80,9 +80,9 @@ func NewCmd(p *print.Printer) *cobra.Command {
 				viper.Set(config.ProjectNameKey, "")
 			}
 
-			err = viper.WriteConfig()
+			err = config.Write()
 			if err != nil {
-				return fmt.Errorf("write new config to file: %w", err)
+				return fmt.Errorf("writing config file: %w", err)
```

Comment:
Related to the comment above about error wrapping, we generally also use the imperative forms of verbs or no verbs at all to help to keep the message concise


```suggestion
				return fmt.Errorf("write config to file: %w", err)
```